crates/statespace-templates/src/starters/snowflake/README.md

```
@@ -0,0 +1,9 @@
+---
+tools:
+  - [snowsql, -c, $SNOWFLAKE_CONNECTION, -q, { regex: "^(SELECT|SHOW|DESCRIBE|EXPLAIN)\\b.*" }]
+---
+
+# Instructions
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)